tests/test_validators.py

```diff
@@ -579,7 +618,7 @@ def test_single_field_uniq_validators(self):
         extra_validators_qty = 2 if django_version[0] >= 5 else 0
         #
```

Comment:
```suggestion
```
"Empty comments don't provide any clarity to the code, and just add clutter. Either add a comment or delete the empty comment."